src/ramanchada2/protocols/calibration.py

```diff
@@ -191,7 +191,12 @@ def _plot(self, ax, **kwargs):
             basefmt=" ",
             label="Reference {}".format(self.sample),
         )
-        ax.set_xlabel("{}".format(self.ref_units))
+
+        if self.ref_units == "cm-1":
+            _units = rf"$\mathrm{{[{_units}]}}$"
```

Comment:
`_units` in the right side of the assignment seems undefined at this point.

Reply:
should be self.ref_units at the right